### PR TITLE
M2P-361 Support Bolt on Magento version 2.1

### PR DESCRIPTION
**Pull request by johnmccombs1**
# Description
Support Bolt on Magento version 2.1

Fixes: https://boltpay.atlassian.net/browse/M2P-350

#changelog M2P-361 Support Bolt on Magento version 2.1

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 
- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Jira ticket link and provided a changelog message.

### CODE REVIEW

File: view/frontend/layout/default.xml [hunk 1]
```diff
@@ -37,12 +37,8 @@
             <block class="Bolt\Boltpay\Block\Js" name="bolt_popup" template="Bolt_Boltpay::popup.phtml" />
         </referenceBlock>
         <referenceContainer name="header-wrapper">
-            <block name="bolt_minicart_additions" as="bolt_minicart_additions"
-                   before="minicart" template="Bolt_Boltpay::checkout/minicart_addons.phtml">
-                <arguments>
-                    <argument name="view_model" xsi:type="object">Bolt\Boltpay\ViewModel\MinicartAddons</argument>
-                </arguments>
-            </block>
+            <block name="bolt_minicart_additions" as="bolt_minicart_additions" class="Bolt\Boltpay\Block\MinicartAddons"
```
Comment (johnmccombs1) [inline view/frontend/layout/default.xml:40]: M2.1 doesn't support View Model

**Review by danac-gs — DISMISSED** [review]
@johnmccombs1 I assume this should not go into master but is a custom branch for HUF. Correct me if I'm wrong. If so, the part related to displaying reward points in minicart could have also been removed / commented out as they don't use that feature. 
LGTM

**Comment by danac-gs** [comment]
There's a conflicting file

**Comment by vitaliyreznikov** [comment]
Yep, this is correct @danac-gs .
@johnmccombs1 could you please change the base branch for this PR?
It should be a new branch like 2.17.0-legacy, the same as 2.17.0.
And the new branch will be changed once we merge this PR.

**Comment by johnmccombs1** [comment]
@vitaliyreznikov @danac-gs Thanks for confirming. Closing this PR

**Comment by johnmccombs1** [comment]
@vitaliyreznikov  @danac-gs This is new PR https://github.com/BoltApp/bolt-magento2/pull/1033

**Pull request closed without merging**